Contentless/Program.cs

```diff
@@ -43,11 +44,64 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Length > 0)
+        {
+            if (args.Length < 2) {
+                Console.WriteLine("You supplied references but there is no project file, this isn't compatible. Please specify the full path of project file, if you want to sync references");
+                return;
+            }
+            var csprojPath = args[1];
+            Console.WriteLine($"Using project file {csprojPath}");
+            var projectRootElement = ProjectRootElement.Open(csprojPath);
+            foreach (var property in projectRootElement.AllChildren.Where(x => x.ElementName == "PackageReference").Select(x => x as ProjectItemElement))
```

Comment:
What about old-style project file `packages.config` references? Are those also handled by this? Sorry, this only just occurred to me haha

Reply:
No, fully ignore this. I have tested.